Source/WebSocket/Websocketpp/x509_cert_utilities.hpp

```diff
@@ -94,8 +98,49 @@ bool verify_cert_chain_platform_specific(asio::ssl::verify_context &verifyCtx, c
         certChain.push_back(std::move(certData));
     }
 
-    auto verify_result = verify_X509_cert_chain(certChain, hostName);
+    auto verify_result = false;
+
+#if HC_PLATFORM == HC_PLATFORM_LINUX
+    X509_STORE *store = X509_STORE_new();
```

Comment:
why isn't this another implementation of `verify_X509_cert_chain`?

Reply:
I placed it there due to already having the cert stack vat to be verified with OpenSSL. My previous implementation had me convert it all back to a cert stack in verify_X509_cert_chain but to prevent errors and to not be redundant I leveraged what was already made. From my understanding of the code what's supposed to happen is that openssl should verify before ever reaching here to this file, but it does not, so we step in here at the very end.

Reply:
we could pass the cert stack down into `verify_X509_cert_chain`
I really want to minimize the amount of code we ifdef for specific platforms as it really hurts readability